### ISSUE TITLE
Add code for empty type checking block

### ISSUE THREAD

**Issue by sondrelg** [snok/flake8-type-checking#28]
I sometimes experience empty type-checking blocks after running cmd+option+o in Pycharm to clean up unused imports.

It would be useful to add a code to flag these, as they "get stuck". They aren't needed, but editors can't tell they're not:

```python
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    pass
```
